tests/libvmi/vmi.go

```diff
@@ -119,12 +118,47 @@ func WithUefi(secureBoot bool) Option {
 // WithSEV adds `launchSecurity` with `sev`.
 func WithSEV() Option {
 	return func(vmi *kvirtv1.VirtualMachineInstance) {
-		vmi.Spec.Domain.LaunchSecurity = &v1.LaunchSecurity{
-			SEV: &v1.SEV{},
+		vmi.Spec.Domain.LaunchSecurity = &kvirtv1.LaunchSecurity{
+			SEV: &kvirtv1.SEV{},
 		}
 	}
 }
 
+func WithVirtioFS(pvcName string) Option {
```

Comment:
would it make sense to make the input variadic ? 

If so, you'd need to make `withVirtioFS` also variadic.

Reply:
I think it would be a nuance to this library

Reply:
I agree that it wouldn't match as well.